### PR TITLE
possessive 

### PR DESCRIPTION
**Pull request by arikrupnik**
Possessive pronouns and contractions are hard. I've taken apostrophes out and added them to resolve confusion between *it's* and *its*.

### CODE REVIEW

**Comment by Hwurzburg** [comment]
Thanks...will review today

**Comment by arikrupnik** [comment]
There are a lot of changes. I double-checked before making the PR, but with so many lines, it's possible that reverse errors crept in.

**Comment by Hwurzburg** [comment]
no change except that now EVERY time WIN10 boots it says that there has been a hardware change and I need to reboot....not an improvement!!
    On Tuesday, August 11, 2020, 08:03:08 AM CDT, arikrupnik <notifications@github.com> wrote:  
 
 


There are a lot of changes. I double-checked before making the PR, but with so many lines, it's possible that reverse errors crept in.

**Comment by Hwurzburg** [comment]
sorry typed on wrong computer screen

File: antennatracker/source/docs/configuration.rst [hunk 1]
```diff
@@ -13,7 +13,7 @@ antenna tracker. Below are the details of this set-up.
 Setting the AHRS orientation
 ============================
 
-If the flight controller board could not be mounted so that it's forward
+If the flight controller board could not be mounted so that its forward
```
Comment (Hwurzburg) [inline antennatracker/source/docs/configuration.rst:16]: incorrect...original is correct

File: antennatracker/source/docs/configuration.rst [hunk 2]
```diff
@@ -116,7 +116,7 @@ do this is to:
 -  Put the slider all the way to the left (if the Reverse checkbox is
    unchecked) or right (if the Reverse checkbox is checked) and push the
    Test button
--  Rotate the tracker so that it's heading (on the MP's Flight Data
+-  Rotate the tracker so that its heading (on the MP's Flight Data
```
Comment (Hwurzburg) [inline antennatracker/source/docs/configuration.rst:119]: orig correct

File: antennatracker/source/docs/frame-assembly.rst [hunk 3]
```diff
@@ -74,6 +74,6 @@ this. :ref:`See the configuration page for more details <configuration_setting_t
 should be taken to ensure the compass is not close to any metal objects.
 
 If using a stand-alone GPS (i.e. not an integrated GPS+compass) the GPS
-should be positioned so that it's antenna has a clear view of the sky.
-It's heading is not important. Some care should be taken to ensure the
+should be positioned so that its antenna has a clear view of the sky.
+Its heading is not important. Some care should be taken to ensure the
```
Comment (Hwurzburg) [inline antennatracker/source/docs/frame-assembly.rst:78]: original correct

File: common/source/docs/common-configuring-a-telemetry-radio-using-mission-planner.rst [hunk 4]
```diff
@@ -64,7 +64,7 @@ Other parameters you may choose to update:
    Latency" if using a joystick or an android tablet's virtual joystick
    to manually fly the vehicle. Note that information on the radio
    signal strength (rssi) and error rate is only sent if this parameter
-   is set to it's default, "MAVLink".
+   is set to its default, "MAVLink".
```
Comment (Hwurzburg) [inline common/source/docs/common-configuring-a-telemetry-radio-using-mission-planner.rst:67]: orig correct

File: common/source/docs/common-configuring-a-telemetry-radio-using-mission-planner.rst [hunk 5]
```diff
@@ -80,7 +80,7 @@ Other parameters you may choose to update:
    transmit time is not generally high. When you set a duty cycle below
    100% then your available bandwidth will be reduced, so you will find
    it will only work well for telemetry at higher air speeds. A radio
-   can be set to receive only by setting it's Duty Cycle to zero.
+   can be set to receive only by setting its Duty Cycle to zero.
```
Comment (Hwurzburg) [inline common/source/docs/common-configuring-a-telemetry-radio-using-mission-planner.rst:83]: original correct

File: common/source/docs/common-electro-permanent-magnet-gripper.rst [hunk 6]
```diff
@@ -14,7 +14,7 @@ The *EPM688* (from NicaDrone.com) is a small (less than 4cm x 4cm x 2cm)
 Electro Permanent Magnet gripper capable of 7kg of force meaning it can
 hold about 1kg of cargo.  The gripping and releasing of the cargo if
 accomplished by delivering a short pulse to an electromagnet which
-reverses it's field.  Objects that it grips onto should be metal or have
+reverses its field.  Objects that it grips onto should be metal or have
```
Comment (Hwurzburg) [inline common/source/docs/common-electro-permanent-magnet-gripper.rst:17]: original correct

File: common/source/docs/common-electro-permanent-magnet-gripper.rst [hunk 7]
```diff
@@ -113,7 +113,7 @@ control through the ground station.
     :target: ../_images/EPM_SetupMissionControlMP.png
 
 Connect with the mission planner. Go to the Config/Tuning >> Full
-Parameters List and set the :ref:`SERVO9_FUNCTION<SERVO9_FUNCTION>` (or RC9_FUNCTION) to "0" (it's default value).
+Parameters List and set the :ref:`SERVO9_FUNCTION<SERVO9_FUNCTION>` (or RC9_FUNCTION) to "0" (its default value).
```
Comment (Hwurzburg) [inline common/source/docs/common-electro-permanent-magnet-gripper.rst:116]: original correct

File: common/source/docs/common-external-magnetometer-for-improved-performance.rst [hunk 8]
```diff
@@ -37,7 +37,7 @@ For the Pixhawk use the :ref:`3DR combination GPS / Magnetometer module <common-
    wires together where possible.
 #. It is also important to keep the magnetometer distanced from magnetic
    metallic objects (use nylon or non magnetic stainless screws and
-   hardware in it's vicinity and use aluminum or nylon standoffs.).
+   hardware in its vicinity and use aluminum or nylon standoffs.).
```
Comment (Hwurzburg) [inline common/source/docs/common-external-magnetometer-for-improved-performance.rst:40]: original correct

File: common/source/docs/common-external-magnetometer-for-improved-performance.rst [hunk 9]
```diff
@@ -55,7 +55,7 @@ For the Pixhawk use the :ref:`3DR combination GPS / Magnetometer module <common-
 -  If the Autopilot board is used in an alternate orientation
    (or upside down), the (:ref:`COMPASS_ORIENT<COMPASS_ORIENT>` ) parameter will need to also
    be adjusted to accommodate the Autopilot boards orientation
-   as well as it's own. (They are additive).
+   as well as its own. (They are additive).
```
Comment (Hwurzburg) [inline common/source/docs/common-external-magnetometer-for-improved-performance.rst:58]: orig correct

File: common/source/docs/common-gripper-servo.rst [hunk 10]
```diff
@@ -31,7 +31,7 @@ Configuration
 - :ref:`GRIP_TYPE <GRIP_TYPE>` to 1 to enable the servo gripper and reboot the autopilot
 - :ref:`GRIP_GRAB <GRIP_GRAB>` to the pwm value (i.e. 1000 ~ 2000) for the gripper's closed position
 - :ref:`GRIP_RELEASE <GRIP_RELEASE>` to the pwm value for the gripper's open position
-- :ref:`GRIP_NEUTRAL <GRIP_NEUTRAL>` to the pwm value for the gripper's neutral position (often the same as it's closed position)
+- :ref:`GRIP_NEUTRAL <GRIP_NEUTRAL>` to the pwm value for the gripper's neutral position (often the same as its closed position)
```
Comment (Hwurzburg) [inline common/source/docs/common-gripper-servo.rst:34]: orig correct

File: common/source/docs/common-gripper-servo.rst [hunk 11]
```diff
@@ -55,4 +55,4 @@ The gripper position can be controlled during a mission in AUTO mode using the :
 Controlling Gripper from the ground station
 ===========================================
 
-The gripper can be opened or closed in real-time from the ground station if the ground station sends a `COMMAND_LONG <https://mavlink.io/en/messages/common.html#COMMAND_LONG>`__ with the command field set to DO_GRIPPER (i.e. 211) but there are no ground stations (as far as we know) that support doing this.
\ No newline at end of file
+The gripper can be opened or closed in real-time from the ground station if the ground station sends a `COMMAND_LONG <https://mavlink.io/en/messages/common.html#COMMAND_LONG>`__ with the command field set to DO_GRIPPER (i.e. 211) but there are no ground stations (as far as we know) that support doing this.
```
Comment (Hwurzburg) [inline common/source/docs/common-gripper-servo.rst:58]: this is no change?

File: common/source/docs/common-here-plus-gps.rst [hunk 12]
```diff
@@ -27,7 +27,7 @@ The Here+ GPS kit comes with a larger round GPS (also knows as the "Rover") that
 
 This GPS is connected to the autopilot using the supplied 8-pin clickmate connector (if using a Cube) or an optional 4pin+6pin DF13 connector (if using Pixhawk).
 
-A smaller GPS (also known as the "base station") labelled "here+" is also included along with it's external antenna.  This GPS is connected to the ground station PC using the supplied USB cable.
+A smaller GPS (also known as the "base station") labelled "here+" is also included along with its external antenna.  This GPS is connected to the ground station PC using the supplied USB cable.
```
Comment (Hwurzburg) [inline common/source/docs/common-here-plus-gps.rst:30]: orig correct

**Comment by peterbarker** [comment]
On Tue, 11 Aug 2020, Henry Wurzburg wrote:

> In antennatracker/source/docs/configuration.rst:
> 
> > @@ -13,7 +13,7 @@ antenna tracker. Below are the details of this set-up.
>  Setting the AHRS orientation
>  ============================
> 
> -If the flight controller board could not be mounted so that it's forward
> +If the flight controller board could not be mounted so that its forward
> 
> incorrect...original is correct

Sorry, but no, the original is incorrect.  The full sentence is:

"If the flight controller board could not be mounted so that it's forward
direction aligned with the antenna"

That contraction expands to:

"If the flight controller board could not be mounted so that it is forward
direction aligned with the antenna"

Which is clearly incorrect.  I mean, there's a second gramatical error in 
the sentence anyway, but inserting "is" will fix that...

"possessive" is appropriate here.

> __________________________________________________________________________________________________________________________________________________________
> 
> In antennatracker/source/docs/configuration.rst:
> 
> > @@ -116,7 +116,7 @@ do this is to:
>  -  Put the slider all the way to the left (if the Reverse checkbox is
>     unchecked) or right (if the Reverse checkbox is checked) and push the
>     Test button
> --  Rotate the tracker so that it's heading (on the MP's Flight Data
> +-  Rotate the tracker so that its heading (on the MP's Flight Data
> 
> orig correct

No, again, the original is incorrect:

"Rotate the tracker so that it's heading (on the MP's Flight Data screen) 
shows close to "0" (i.e. North)"

That should be using the possessive, "its".

Just in case:

  "it's" is simply short for "it is".

  "its" is the possessive form for "it".  Think of this in terms of the 
possessive form for "he" - we do not say, "He ate he's icecream" - we have 
a special possessive form, "his", so we get "He ate his icecream".  The 
gender-neutral equivalent is "It ate its icecream".

English.  Gotta love it.

**Comment by arikrupnik** [comment]
> Sorry, but no, the original is incorrect. The full sentence is: "If the flight controller board could not be mounted so that it's forward direction aligned with the antenna" That contraction expands to: "If the flight controller board could not be mounted so that it is forward direction aligned with the antenna" Which is clearly incorrect. I mean, there's a second gramatical error in the sentence anyway, but inserting "is" will fix that... "possessive" is appropriate here.

In this PR, I limited myself to resolving possessive *its* and the contraction *it's*. I agree that the rest of this sentence can use improvement. Perhaps it can use the active voice, "so that its forward direction _aligns_ with the antenna."

**Comment by Hwurzburg** [comment]
I'm obviously tired and someone else review this...sorry...taking a break for a while...

File: antennatracker/source/docs/configuration.rst [hunk 13]
```diff
@@ -13,7 +13,7 @@ antenna tracker. Below are the details of this set-up.
 Setting the AHRS orientation
 ============================
 
-If the flight controller board could not be mounted so that it's forward
+If the flight controller board could not be mounted so that its forward
 direction aligned with the antenna (as mentioned on the :ref:`Assembling a frame page <frame-assembly>`) the AHRS_ORIENTATION parameter should
```
Comment (Hwurzburg) [inline antennatracker/source/docs/configuration.rst:17]: could you change "direction aligned" to "direction is aligned"? might as well fix that also

File: common/source/docs/common-gripper-servo.rst (hunk 11 above)
Comment (Hwurzburg) [reply common/source/docs/common-gripper-servo.rst:58]: np

File: antennatracker/source/docs/configuration.rst (hunk 13 above)
Comment (Hwurzburg) [reply antennatracker/source/docs/configuration.rst:17]: reveiwed again, going to merge now and quick fix myself..thanks for the contribution

File: common/source/docs/common-gripper-servo.rst (hunk 11 above)
Comment (arikrupnik) [reply common/source/docs/common-gripper-servo.rst:58]: I think the diff is in whitespace, one file ends on a character and the other has a newline at the end.

File: antennatracker/source/docs/configuration.rst (hunk 13 above)
Comment (arikrupnik) [reply antennatracker/source/docs/configuration.rst:17]: You were too fast for me! Let me know if there's anything else you want me to fix.

**Pull request merged**